resources/views/emails/invitation.blade.php

```diff
@@ -0,0 +1,9 @@
+@component('mail::message')
+<h3>Hello {{$receiver}}</h3>
```

Comment:
```suggestion
<h3>Hello {{$receiver}}</h3>
```
I see `$receiver ` is the email, maybe here nice will be show name not email.

Reply:
it would be nice if the name of the invited user was known
The inviting user input only email of the invited user, so we don't know his name yet.